src/ui/pages/AssetInfo/AssetInfo.tsx

```diff
@@ -86,6 +70,34 @@ function ReportAssetLink({ asset }: { asset: Asset }) {
   );
 }
 
+function ShareAssetLink({ asset }: { asset: Asset }) {
+  const toastRef = useRef<PopoverToastHandle>(null);
+  const { handleCopy } = useCopyToClipboard({
+    text: `https://app.zerion.io/tokens/${asset.symbol}-${asset.id}`,
```

Comment:
Just question, I'm not sure, but shouldn't it be like this?
```
https://app.zerion.io/tokens/${asset.symbol}-${asset.asset_code}
```
I mean using `asset_code` for the token URL's suffix instead of `id`

Reply:
Rest API asset doesn't have asset_code anymore. In positions we had a convert function to transform new Asset into old one. This should be fixed soon and new object should be used everywhere.